examples/ConnectedInteractiveChart.tsx

```diff
@@ -23,6 +23,7 @@ import {
   // This is one such action creator.
   setXDomain
 } from '../src';
+import { ChartState } from '../src/connected/model/state';
```

Comment:
`ChartState` is not re-exported -- do you think it makes sense to expose `ChartState`?